ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-stable-4.14-upgrade-from-stable-4.13.yaml

```diff
@@ -81,6 +81,26 @@ tests:
     test:
     - chain: openshift-upgrade-qe-sanity-disconnected
     workflow: cucushift-installer-rehearse-aws-ipi-disconnected-private-cco-manual-security-token-service-private-s3-with-ep-sts-ec2-elb
+- as: aws-ipi-usertags-custom-sg-fips-f28
```

Comment:
We don't need to add too many stable - stable job, 1 platform has 1 job is enough.